module_auto_update/__manifest__.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Module Auto Update",
+    "summary": "Automatically update Odoo modules",
+    "version": "10.0.1.0.0",
+    "category": "Uncategorized",
+    "website": "https://odoo-community.org/",
+    'author': 'Brent Hughes, '
```

Comment:
`LasLabs` if on the clock. You go in Contributors section of ReadMe using LasLabs email.